qualtran/bloqs/arithmetic/addition_test.py

```diff
@@ -272,7 +272,7 @@ def test_add_mod_n(bitsize, mod, add_val, cvs):
     op = gate.on_registers(**get_named_qubits(gate.signature))
     circuit = cirq.Circuit(op)
     cirq.testing.assert_equivalent_computational_basis_map(basis_map, circuit)
-    circuit += op**-1
+    circuit += cirq.inverse(op)
```

Comment:
does this work? I think this is a bigger change than you think

Reply:
Why do you say so?  I think it does the same thing in a slightly safer (and more type friendly) way.

`"Returns the inverse `val**-1` of the given value, if defined"
`
https://github.com/quantumlib/Cirq/blob/v1.3.0/cirq-core/cirq/protocols/inverse_protocol.py#L70-L121